tests/monotouch-test/UIKit/UIContentSizeCategoryTest.cs

```diff
@@ -45,6 +45,17 @@ public void Compare ()
 			Assert.Throws<ArgumentException> (() => UIContentSizeCategoryExtensions.Compare (small, (UIContentSizeCategory)271828));
 			Assert.Throws<ArgumentException> (() => ((UIContentSizeCategory)1234).IsAccessibilityCategory ());
 		}
+
+		[Test]
+		public void GetPreferredContentSizeCategoryTest ()
+		{
+			var sizeNSString = UIApplication.SharedApplication.PreferredContentSizeCategory;
+            var sizeEnum = UIContentSizeCategoryExtensions.GetValue(sizeNSString);
+            var size = UIApplication.SharedApplication.GetPreferredContentSizeCategory();
+            Assert.True (sizeEnum == size, "String");
```

Comment:
It's often better to use the Assert.AreEqual method instead:

```suggestion
            Assert.AreEqual (sizeEnum, size, "String");
```

because there's a difference when the assert fails.

In your case, the failure will be reported as something like "expected true, got false", which isn't very helpful, because as you start diagnosing what's wrong, the first question you'll ask yourself is if the expected value is wrong (maybe iOS changed its behavior, and we need to update the test), or if the actual value is wrong (might be due to a bug in our code).

If you instead use the Assert.AreEqual method, the failure will be reported as something like "expected UIContentSizeCategory.Medium, got UIContentSizeCategory.Small", which contains enough information to answer that first question from above, and you save a lot of time when debugging test failures.